fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/codec/LuceneOptimizedPostingsFormat.java

```diff
@@ -36,11 +41,16 @@
 /**
  * {@code PostingsFormat} optimized for FDB storage.
  */
+@AutoService(PostingsFormat.class)
 public class LuceneOptimizedPostingsFormat extends PostingsFormat {
     PostingsFormat postingsFormat;
 
+    public LuceneOptimizedPostingsFormat() {
```

Comment:
Same as above - is this new constructor necessary?

Reply:
Yes..  same as above...